packages/cannon-worker-api/src/worker/index.ts

```diff
@@ -113,6 +113,14 @@ self.onmessage = ({ data }: { data: CannonMessage }) => {
     case 'setFixedRotation':
       state.bodies[data.uuid].fixedRotation = data.props
       break
+    case 'setFrictionGravity':
+      if (!data.props) {
+        // The user may reset frictionGravity.
+        delete state.world.frictionGravity
+        return;
+      }
+      state.world.frictionGravity = new Vec3(data.props[0], data.props[1], data.props[2])
```

Comment:
I'm not real keen on having the return in the middle of this switch statement.

```suggestion
      state.world.frictionGravity = data.props ? new Vec3(...data.props) : data.props
```